README.md

```diff
@@ -1,16 +1,82 @@
-# **Stock Market**
+# **Stock Market Project**
 
 ## Project summary
-In this project I created a system which simulates a stock market. The stock market composed of some companies. The system enables the user to insert and remove companies from the stock market and to make different actions.
+In this project I simulated stock market using two classes. The stock market class represents the entire set of companies that are found in the stock market and enables the user to conduct a variety of actions the different companies present in the market. By using the API given by the classes, different manipulations can be conducted such as changing net worth of companies, merging two companies and more. The company class represents a specific company and enables to edit parameters that charcterize the company. 
 
-##  About the Classes
-Two classes created to implement the system idea:
+##  Project Design
 
-1. *Company* - This class includes the required methods and represents a specific company.
+### Company class
+The company class is described using the following fields: 
+* **name** - represents the name of the company. The type of this field is string. 
+* **stocks_num** - represents the number of company's stocks. The type of this field is integer. 
+* **stock_price** - represents the price of one stock. The type of this field is integer or float.
+* **comp_type** - represents the type of the company. For example: High Tech, Pharmaceutical, Hardware, etc. The type of this field is string. 
 
-2. *StockMarket* - This class represents the whole stock market which contains a list of companies.
+The company class API includes the following functions:
+
+* *net_worth* - calculates the company's net worth. 
+
+* *set_name* - updates the name of the company
+
+* *set_stocks_num* - updates the number of company's stocks. 
+
+* *set_stocks_price* - updates the price of company's stock.
+
+* *set_comp_type* - updates the company type.
```

Comment:
Usually when describing function, the best possible description would be what are the inputs and what are the outputs/prints, adittional information can be how is the output generated. for example, the function receives a new stock price and sets the stock count accordingly (in this case, there is an input value and no output).

Reply:
What about functions that only gets self?